### ISSUE TITLE
"Stop" Button Closes *All* Open Games (When Using Multiple IDEs)

### ISSUE THREAD

**Issue by TheGitMagician** [YoYoGames/GameMaker-Bugs#2096]
### Description

If you work with multiple IDEs (e.g. during networking tests with a server and a client) then pushing the "Stop" button in one IDE closes *all* open games instead of just the one that's associated with the IDE.

I just want to make sure whether this is intended behavior or not.

I stubled across this when I worked with a Windows server in one IDE and an HTML5 client in another IDE. I usually don't close the browser tab of the HTML5 client between runs but instead push the "Stop" button and then run the HTML5 client again. This causes the Windows server - which has been opened from the other IDE - to close as well.

### Expected Change

The "Stop" button only closes the game that has been run from that IDE.

### Steps To Reproduce

1. Start two IDEs and run the game in each.
2. Press the "STOP" button in one of the IDEs.
3. Both games will close.

### How reliably can you recreate this issue using your steps above?

Always

### Which version of GameMaker are you reporting this issue for?

2023.8.2 (Monthly)

### Which operating system(s) are you seeing the problem on?

Windows 10

### Are you running GameMaker from inside your Steam library?

No

### Contact Us Package Attached?

- [X] I have attached my Contact Us Package

### Sample Project Added?

- [ ] I have included a small sample project

**Comment by gnysek** [comment]
Yeah, that's frustrating, I've noticed that when I was working on game which got separate client and server.
